dist/autosize.js

```diff
@@ -109,10 +111,14 @@
 			var arr = [];
 
 			while (el && el.parentNode && el.parentNode instanceof Element) {
-				if (el.parentNode.scrollTop) {
+				var scrollTop = el.parentNode.scrollTop;
+				if (el.parentNode === document.body) {
+					scrollTop = document.documentElement.scrollTop;
```

Comment:
this is the fix for the mobile problem. Basically body.scrollTop doesn't work as expected in some browsers, like FF. My best guest is that the latest webkit on safari is using something similar.

The library was already using something like it, but they forgot this.